_data/sites.json

```
@@ -16231,11 +16231,10 @@
 
     {
         "name": "Newegg",
-        "email": "privacy@newegg.com",
-        "url": "https://kb.newegg.com/knowledge-base/privacy-policy-newegg/#12-what-are-your-rights",
-        "difficulty": "limited",
-        "notes": "Deleting your account is only possible if an applicable government policy gives you the right to delete your account.",
-        "notes_de": "Du kannst deinen Account nur löschen, wenn die Gesetze deines Landes dir das Recht dazu geben.",
+        "url": "https://kb.newegg.com/knowledge-base/data-rights-request/",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
Still sounds like they have someone do it manually, so I'd say it is hard.